liberbatch.py

```diff
@@ -1,19 +1,15 @@
 import os, meichecker, liberupdatev5
+PATH = "your path here"
 
-directory = os.fsencode("C:\\Users\\cole_\\Downloads\\liberUsualis - Copy\\liber")
+directory = os.fsencode(PATH)
 err=""
 
-i = 0 #for debugging
 for mefile in os.listdir(directory):
     filename = os.fsdecode(mefile)
     if filename.endswith("corr.mei"):

```

Comment:
This looks like a requirement (that the files end with `corr.mei`)! This should be documented somwhere.